recipe/meta.yaml

```diff
@@ -16,9 +16,9 @@ build:
   number: 0
   script: {{ PYTHON }} -m pip install . -vvv --no-deps --no-build-isolation
   script_env:
-    - MAX_JOBS=$CPU_COUNT
-    # Not compiling for 8.0;8.6;8.9;9.0+PTX to keep builds under 6 hours
-    - TORCH_CUDA_ARCH_LIST=8.0+PTX
+    # Limit MAX_JOBS in order to prevent runners from crashing
+    - MAX_JOBS=1
```

Comment:
> On the openstack runners, the jobs are crashing during the first compilation phase. Trying to reduce the number of concurrent compilations to 2 in order to prevent this crash (from a max of 4).

Looking at https://github.com/Quansight/open-gpu-server?tab=readme-ov-file#cpu-runners:

- `cirun-openstack-cpu-large` has 4vCPUs, 12GB of RAM
- `cirun-openstack-cpu-xlarge` has 4vCPUs, 32GB of RAM

Should we maybe use `xlarge` instead with more RAM?

Reply:
If you know how to switch runners, then do it!

Reply:
We might need to submit another request to https://github.com/conda-forge/admin-requests first? Let's wait a day see how the current build goes, want to see how close/far we are away from building this successfully with the current setup.